Exiled.Events/Handlers/Scp330.cs

```diff
@@ -0,0 +1,42 @@
+// -----------------------------------------------------------------------
+// <copyright file="Scp330.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Handlers
+{
+    using Exiled.Events.EventArgs;
+    using Exiled.Events.Extensions;
+
+    using static Exiled.Events.Events;
+
+    /// <summary>
+    /// SCP330 related events.
+    /// </summary>
+    public static class Scp330
+    {
+        /// <summary>
+        /// Invoked before player ate SCP330.
+        /// </summary>
+        public static event CustomEventHandler<EatingScp330EventArgs> Eating;
```

Comment:
I'd change to `EatingScp330`

Reply:
Not to mention fixing the past-tense version of "eat" in future-tense event summary.

Reply:
I mentioned everything Joker. Don't worry.

Reply:
I'm german sorry.